x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -840,4 +840,26 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'mac.advanced.kernel.network_extension.enable_content_filtering',
+    first_supported_version: '8.1',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.kernel.network_extension.enable_content_filtering',
+      {
+        defaultMessage:
+          'Enable or disable the network content filter, this will enable/disable network eventing, additionally host isolation will fail if this option is disabled. Default: true',
```

Comment:
I think that comma after `...content filter...` is really suppose to be a period, no? The `additionally...` right after the comma also is not sounding right to me